scale/job/seed/manifest.py

```diff
@@ -207,7 +207,7 @@ def get_input_interface(self):
         """
 
         input_dict = self.get_inputs()
```

Comment:
remove unused input_dict?

Reply:
I made this change at the request of Derick.  @dfaller any thoughts?